make.bat

```diff
@@ -18,7 +18,7 @@ goto :eof
 
 :.dapper
 if not exist .dapper.exe (
-    bitsadmin.exe /rawreturn /transfer dappwer-download https://releases.rancher.com/dapper/latest/dapper-Windows-x86_64.exe %~dp0\.dapper.exe
+    bitsadmin /rawreturn /transfer dappwer-download https://releases.rancher.com/dapper/latest/dapper-Windows-x86_64.exe %~dp0\.dapper.exe
```

Comment:
the `exe` needed to be dropped?

Reply:
Doesn't need to be dropped, but matches what we have in wmi_exporter-package https://github.com/rancher/wmi_exporter-package/blob/8e0c309b986cfbfb96ec1dba0a40df612f149f36/make.bat#L25